requestor/erigolem_cli.py

```diff
@@ -75,5 +224,28 @@ def stop_instance(client: ErigolemClient, instance_id: str):
     click.echo(instance)
 
 
+@cli.command(name='net-check')
+@click.option('--num-instances', type=click.INT, default=10, envvar='NUM_INSTANCES')
+@click.option('--pending-timeout-sec', type=click.INT, default=60, envvar='PENDING_TIMEOUT_SEC')
+@click.option('--check-interval-sec', type=click.INT, default=60, envvar='CHECK_INTERVAL_SEC')
+@click.option('--init-params', default=DEFAULT_INIT_PARAMS, envvar='INIT_PARAMS')
+@click.pass_obj
+def net_check(
+        client: ErigolemClient,
+        num_instances: int,
+        pending_timeout_sec: int,
+        check_interval_sec: int,
+        init_params: str
+):
+    checker = NetworkChecker(
+        client=client,
+        num_instances=num_instances,
+        pending_timeout_sec=pending_timeout_sec,
+        check_interval_sec=check_interval_sec,
```

Comment:
These don't seem to me keyword args - so maybe pass them directly (without `x=x`)?

Reply:
I believe it's a good practice to use keyword arguments when there's more than 1-2 parameters, because it's much less error-prone. Reviewing this code you didn't have to check the function signature twice to make sure that argument order is right, have you?